src/main/kotlin/com/sourcegraph/cody/chat/ui/MessagesPanel.kt

```diff
@@ -43,6 +49,32 @@ class MessagesPanel(private val project: Project) :
     repaint()
   }
 
+  @RequiresEdt
+  @Synchronized
+  fun addOrUpdateContext(message: ChatMessage) {
+    removeBlinkingCursor()
+    createUsedContextFilesPanel(message)?.let(::addComponentToChat)
+    add(BlinkingCursorComponent.instance)
```

Comment:
I remove the blinking cursor and re-add it just after the context files panel. We receive context files much faster than the actual response. Trust me, it looks good now.